akka-cluster-tools/src/main/scala/akka/cluster/client/ClusterClient.scala

```diff
@@ -171,6 +173,63 @@ final class ClusterClientSettings(
       heartbeatInterval, acceptableHeartbeatPause, bufferSize, reconnectTimeout)
 }
 
+/**
+ * Declares a super type for all events emitted by the `ClusterClient`
+ * in relation to contact points being added or removed.
+ * Events are published to the Akka event stream (system.eventStream).
+ * Given that the event bus is local only, there is no serialization
+ * of them that is required.
+ */
+sealed trait ContactPointChange extends NoSerializationVerificationNeeded {
+  val contactPoint: ActorSelection
```

Comment:
It is held as a selection internally (before I got to this PR)...

I can make it a path though if you'd prefer. Please provide the rationale also, so that I can understand for the future. Thanks.